gui/src/main/java/org/jboss/as/console/client/core/ReadRequiredResources.java

```diff
@@ -62,6 +64,7 @@
     private final DispatchAsync dispatcher;
     private final StatementContext statementContext;
     private final List<Input> input;
+    private String locale = Preferences.get(Preferences.Key.LOCALE, "en");
```

Comment:
`Preferences.get(Preferences.Key.LOCALE)` might return null. I suggest to replace this with `Preferences.get(Preferences.Key.LOCALE) != null ? Preferences.get(Preferences.Key.LOCALE) : "en"`